dashboard/src/main/resources/templates/macros.ftl

```diff
@@ -0,0 +1,81 @@
+<#macro formatJarLinkageReport jarLinkageReport jarToDependencyPaths>
```

Comment:
This file is introduced to share `formatJarLinkageReport` macro among conponent.ftl and dashboard.ftl.